### PR TITLE
slimes now spawn two more slimes on death

### PR DESCRIPTION
**Pull request by brandonhutchins**

### CODE REVIEW

**Review by tpayne1 — APPROVED** [review]

File: source/scripts/model/Monster.js [hunk 1]
```diff
@@ -116,9 +118,7 @@ export default class Monster {
         this.health -= damage
         if(this.health <= 0) {
             this.game.remove("monsters", this)
-            if(this.basesprite == DATA.SPRITES.MONSTERS.BLUE_SLIME) {
-                console.log("yay you killed a blue lime")
-            }
+            this.onDeath()
```
Comment (ehgoodenough) [inline source/scripts/model/Monster.js:127]: Nice.

File: source/scripts/model/Monster.js [hunk 2]
```diff
@@ -31,6 +32,7 @@ export default class Monster {
         this.turnCounter = monster.protomonster.turnCounter || function () {
             this.phase = !this.phase
         }
+        this.onDeath = monster.protomonster.onDeath || function () {}
```
Comment (ehgoodenough) [inline source/scripts/model/Monster.js:37]: Nice nice.

**Review by ehgoodenough — APPROVED** [review]
Solid code. Good execution. Give me a playable build to test, and I'll merge it.

**Comment by brandonhutchins** [comment]
https://mocsarcade.github.io/enchiridion/SlimeSpawnTest

**Pull request merged**